rust/query/typeql_match.rs

```diff
@@ -138,46 +137,51 @@ pub struct Filter {
 
 impl fmt::Display for Filter {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        write!(f, "{} ", Get)?;
+        write!(f, "{} ", token::Filter::Get)?;
         write_joined!(f, ", ", self.vars)
     }
 }
 
-#[derive(Clone, Debug, Eq, PartialEq)]
-pub struct OrderedVariable {
-    pub var: UnboundVariable,
-    pub order: Option<String>, // FIXME
-}
-
-impl OrderedVariable {
-    fn new(var: UnboundVariable, order: &str) -> Self {
-        OrderedVariable {
-            var,
-            order: match order {
-                "" => None,
-                order => Some(order.to_string()),
-            },
+pub mod sorting {
+    use crate::pattern::UnboundVariable;
+    use std::fmt;
+
+    #[derive(Clone, Debug, Eq, PartialEq)]
+    pub struct OrderedVariable {
+        pub var: UnboundVariable,
+        pub order: Option<String>, // FIXME
+    }
```

Comment:
Allows us to specify `OrderedVariable` => `sorting::OrderedVariable`, to set it apart from all the other `FooVariable`s we have.